src/components/Portfolio/LoanList.vue

```diff
@@ -0,0 +1,245 @@
+<template>
+	<div class="tw-mt-4">
+		<div class="tw-relative">
+			<div class="tw-overflow-x-auto tw-min-w-full">
+				<table class="tw-w-full">
+					<thead>
+						<tr class="tw-border-y tw-border-tertiary">
+							<th class="tw-text-left tw-font-medium tw-px-2 tw-py-1">
+								Loan details
+							</th>
+							<th class="tw-text-left tw-font-medium tw-px-2 tw-py-1">
+								Status
+							</th>
+							<th class="tw-text-right tw-font-medium tw-px-2 tw-py-1">
+								You loaned
+							</th>
+							<th class="tw-text-right tw-font-medium tw-px-2 tw-py-1">
+								Amount
+							</th>
+							<th class="tw-text-right tw-font-medium tw-px-2 tw-py-1">
+								Length
+							</th>
+							<th class="tw-text-right tw-font-medium tw-px-2 tw-py-1">
+								Paid back or raised
+							</th>
+							<th class="tw-text-left tw-font-medium tw-px-2 tw-py-1">
+								Team
+							</th>
+						</tr>
+					</thead>
+					<tbody>
+						<tr v-if="loading">
+							<td colspan="7" class="tw-px-2 tw-py-4">
+								<div v-for="i in 3" :key="i" class="tw-grid tw-grid-cols-12 tw-gap-4 tw-mb-4">
+									<kv-loading-placeholder
+										v-for="(placeholder, index) in placeholders"
+										:key="index"
+										:class="[
+											`tw-col-span-${placeholder.span}`,
+											placeholder.marginLeft && 'tw-ml-auto'
+										]"
+										style="height: 16px;"
+									/>
+								</div>
+							</td>
+						</tr>
+						<tr v-else-if="!loans.length">
+							<td class="tw-text-center tw-text-secondary tw-px-2" colspan="7">
+								No loans found
+							</td>
+						</tr>
+						<tr
+							v-for="loan in loans"
+							:key="loan.id"
+							class="tw-border-b tw-border-tertiary"
+						>
+							<td class="tw-px-2 tw-py-2">
+								<div class="tw-flex tw-items-center">
+									<img
+										:src="loan.image.url"
+										alt="Loan image"
+										class="tw-w-[50px] tw-h-[50px] tw-mr-2"
```

Comment:
We try to avoid TW dynamic classes, since it adds to the compiled CSS file size, so it would be good to just inline styling or a class for these two dynamic classes.